cli/tests/pcluster/schemas/test_imagebuilder_schema/test_imagebuilder_schema/imagebuilder_schema_dev.yaml

```diff
@@ -54,4 +54,10 @@ DevSettings:
   DisableValidateAndTest: True
   DisableKernelUpdate: True
 
-CustomS3Bucket: bucket-name
\ No newline at end of file
+DeploymentSettings:
+  {% if disable_sudo_access_for_default_user == "True" %}DisableSudoAccessForDefaultUser: True{% endif %}
```

Comment:
*Minor*
In general, it's better to always inject the optional parameter if defined, even if it is injected with the default value because in that way the config is immediately extensible to cover both cases.

Example:

```
{% if property_variable %}
  PropertyName: {{ property_variable }}
{% endif %}
```

Reply:
The Image Config Should not be provided with `DisableSudoAccessForDefaultUser` at all which is the Unknown field as we changed the schema.

So, want to this field for Negative testing